webrtc.html

```diff
@@ -6530,8 +6555,17 @@ <h2>Attributes</h2>
               When one of the SSRCs for RTP source media streams received
               by <var>receiver</var> is removed (either due to reception of
               a BYE or via timeout), the <code>mute</code> event is fired at
-              <code>track</code>.  If and when packets are received again,
-              the <code>unmute</code> event is fired at <code>track</code>.</p>
+              <code>track</code>. <a data-lt="set the RTCSessionDescription">
+              Setting an <code>RTCSessionDescription</code></a> can also
+              lead to the firing of the <code>mute</code> event at
+              <code>track</code>.
+              If and when packets are received again, and either the
+              [[\PendingDirection]] or [[\CurrentDirection]] slot of the
+              <a><code>RTCRtpTransceiver</code></a> the
+              <a><code>RTCRtpReceiver</code></a> is member of is either 
+              <code>"recvonly"</code> or <code>"sendrecv"</code>, the
+              the <code>unmute</code> event is fired at <code>track</code>.
```

Comment:
If it was muted by a BYE, an out of order packet that comes after the BYE should not unmute it.

Reply:
That part is unchanged, filed #1748 to track resolution.